python/eos/plot/plotter.py

```diff
@@ -76,6 +76,8 @@ def setup_plot(self):
         mytitle = ''
         myylabel = ''
         myxlabel = ''
+        myyscale = 'linear'
```

Comment:
It would be clearer what are possible and default values for `myXscale` if the default value was not assigned and then overriden as an option, but instead assigned if the argument is not supplied by the user via an `else` below.

Reply:
True

Reply:
On the other hand, it wouldn't work with the current code since labels and scales need to be defined even if the `x` or the `y` argument are not there